aries-site/src/layouts/content/Example/ExampleControls.js

```diff
@@ -28,10 +28,10 @@ export const ExampleControls = ({
 
   const boxProps = !horizontalLayout
     ? {
-        background: 'background-front',
+        background: { color: 'background-contrast', opacity: 0.02 },
```

Comment:
Where does the opacity come from? What does it enable?

Add code comment?

Reply:
I've made a design proposal for the dark mode treatment in my latest commit and added code comments. LMK your thoughts.